.ci/setup_env_centos.sh

```diff
@@ -18,6 +18,8 @@ else
   centos_version=$(grep VERSION_ID /usr/lib/os-release | cut -d '"' -f2)
 fi
 
+sudo -E yum -y clean all
```

Comment:
Do we know why it fixes it? Given afaik all the centos CIs will be running in clean deploys, it seems odd.

Reply:
not really, I am trying to implement what I see that has been suggested as a solution on the CentOS forums